dvc/command/experiments.py

```diff
@@ -308,44 +321,57 @@ def _experiments_table(all_experiments, **kwargs):
         include_params=include_params,
         exclude_params=exclude_params,
     )
+    metric_headers = _normalize_headers(metric_names)
+    param_headers = _normalize_headers(param_names)
 
-    table = Table()
-    table.add_column(
-        "Experiment", no_wrap=True, header_style="black on grey93"
-    )
-    if not kwargs.get("no_timestamp", False):
-        table.add_column("Created", header_style="black on grey93")
-    _add_data_columns(
-        table,
+    td = experiments_table(
+        all_experiments,
+        metric_headers,
         metric_names,
-        justify="right",
-        no_wrap=True,
-        header_style="black on cornsilk1",
-    )
-    _add_data_columns(
-        table, param_names, justify="left", header_style="black on light_cyan1"
-    )
-
-    for base_rev, experiments in all_experiments.items():
-        for row, _, in _collect_rows(
-            base_rev, experiments, metric_names, param_names, **kwargs,
-        ):
-            table.add_row(*row)
```

Comment:
@skshetry the confusion about the row style thing is that this was originally
```
for row, style in _collect_rows(...):
    ....
    table.add_row(*row, style=style)
```
it got turned off at some point during the pre-release testing phase due to a terminal compatibility bug and was never turned back on (by mistake). It should be turned back on now.

Reply:
see also: https://github.com/iterative/dvc/pull/5829/files#diff-150402093379363d315fa80a2af5b8576a883a197951aeb66228e2a459630ec7